### ISSUE TITLE
ifs != NULL  patch

### ISSUE THREAD

**Issue by rain-1** [Enigma-Game/Enigma#14]
This did not compile. Here is a fix:

```
diff -Naur enigma-1.21-a/src/lev/Proxy.cc enigma-1.21-b/src/lev/Proxy.cc
--- enigma-1.21-a/src/lev/Proxy.cc	2014-12-19 21:24:37.000000000 +0000
+++ enigma-1.21-b/src/lev/Proxy.cc	2019-01-08 22:55:39.508341399 +0000
@@ -933,10 +933,10 @@
                     } else if (haveLocalCopy) {
                         // plain file
                         basic_ifstream<char> ifs(absExtPath.c_str(), ios::binary | ios::in);
-                        if (ifs != NULL)
-                            Readfile(ifs, extCode);
-                        else
+                        if (!ifs)
                             haveLocalCopy = false;
+                        else
+                            Readfile(ifs, extCode);
                     }
 
                     if (!haveLocalCopy) {

```

**Comment by raoulb** [comment]
Thanks for reporting. I think this was fixed on master with commit e0f31f63ba4252ced7128d3ceec2bc668e8681f4.

**Comment by rain-1** [comment]
cheers!

Bug affects release 1.21 and 1.20 on the download page.
